spec/integration/connection_spec.rb

```diff
@@ -17,6 +17,9 @@
     end
 
     `gpg --import #{KEYS.join(' ')}`
+
+    vcr_mode_none = ENV['VCR_ON'] == 'true' && (!ENV['VCR_RECORD_MODE'] || ENV['VCR_RECORD_MODE'] == 'none') # TODO: refactor
```

Comment:
error: Identical code found in global_helpers.rb:234, connection_spec.rb:21 (mass = 68) (flay)